src/coreclr/gc/gc.cpp

```diff
@@ -32965,6 +33137,12 @@ void gc_heap::plan_phase (int condemned_gen_number)
         x = find_next_marked (x, end, use_mark_list, mark_list_next, mark_list_index);
     }
 
+#ifdef USE_REGIONS
+#ifndef MULTIPLE_HEAPS
+    free_list_snapshot::record(plan_should_sweep_start, free_regions_basic, &free_regions[(int)basic_free_region]);
```

Comment:
give it a different stage name here? eg, after_plan.

Reply:
good catch - I had an enum entry for it but didn't use it after copying the call